### DISCUSSION TITLE
Firefox Multi-Account Containers

### DISCUSSION THREAD

**Discussion by QasimK** [QasimK/qasimk.github.io#29] [Blog Posts]
Comment on [Firefox Multi-Account Containers](http://qasimk.io/2017/firefox-containers/).

**Comment by gimibear** [comment]
What about antidetect browsers like gologinapp and others? Good alternative?

**Comment by QasimK** [comment]
That sounds interesting; unfortunately I don't have any experience with them to be able to say.
